app/models/item_order.rb

```diff
@@ -9,4 +9,12 @@ class ItemOrder <ApplicationRecord
   def subtotal
     price * quantity
   end
+
+  def is_mine?(merchant_id)
```

Comment:
This should go in item or merchant because those are the resources this actually tests.